spring-kafka/src/test/java/org/springframework/kafka/listener/KafkaMessageListenerContainerTests.java

```diff
@@ -237,7 +243,7 @@ public void testSlowConsumerCommitsAreProcessed() throws Exception {
 		// Verify that commitSync is called when paused
 		assertThat(latch.await(60, TimeUnit.SECONDS)).isTrue();
 		verify(consumer, atLeastOnce()).pause(any(TopicPartition.class), any(TopicPartition.class));
-		verify(consumer, atLeastOnce()).commitSync(any());
+		verify(consumer, atMost(2)).commitSync(any());
```

Comment:
Missed this on the first pass, unfortunately. Why are we doing this, as it completely changes the semantic of the test? It would pass if no commits are made, and why is there an upper bound of the number of commit operations?